adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -419,6 +421,13 @@ - (NSString *)getSuperToken {
   return [VungleSDK.sharedSDK currentSuperToken];
 }
 
+- (void)setCOPPAStatus:(NSNumber *)coppa {
```

Comment:
Ditto, shouldn't this take in boolean instead of number?
https://developers.google.com/admob/ios/api/reference/Classes/GADRequestConfiguration?hl=en#-tagforchilddirectedtreatment: